Changes to CZML, clean up appearance

ntariq opened the pull request.

Commit:
Put all port labels/values in single blocks per port

Commit:
Removing power line images, adding Power Line label

marionthefourth reviewed (changes requested):
Can you rename the elements with the following IDs:

- 758c944f-3278-4f5c-b141-ec88263f2e63 -> "Financial/Ship Information"
- 2e176e1a-81d4-4269-8588-826d6ef026f6 -> "Gulangyu Sensor Air Quality"
- bead2d08-5ded-41e6-b583-a835bec91cbc -> "Xiamen Sensor Air Quality"
- 838d65b6-8a86-41b0-af9a-f5a477a3ee6e -> "Hongwen Sensor Air Quality"

Commit:
Updated Financial/Ship and Port Sensor Air Quality Element Names

758c944f-3278-4f5c-b141-ec88263f2e63 -> "Financial/Ship Information"
2e176e1a-81d4-4269-8588-826d6ef026f6 -> "Gulangyu Sensor Air Quality"
bead2d08-5ded-41e6-b583-a835bec91cbc -> "Xiamen Sensor Air Quality"
838d65b6-8a86-41b0-af9a-f5a477a3ee6e -> "Hongwen Sensor Air Quality"
